apps/docs/pages/guides/auth/social-login/auth-linkedin.mdx

```diff
@@ -122,7 +122,7 @@ suspend fun signOut() {
 
 ## LinkedIn Open ID Connect (OIDC)
 
-To support recent changes to the LinkedIn [OAuth APIs](https://learn.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow?context=linkedin%2Fcontext&tabs=HTTPS1), we will be replacing the _LinkedIn_ provider with a new _LinkedIn (OIDC)_ provider which utilizes the [Open ID Connect standard](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin-v2#validating-id-tokens). Consequently, we have disabled edits on the _LinkedIn_ provider and will be removing it effective 30th November 2023. Developers with LinkedIn OAuth Applications created prior to 1st August 2023 should create a new OAuth application [via the steps outlined above](/docs/guides/auth/social-login/auth-linkedin#create-a-linkedin-oauth-app) and migrate their credentials from the _LinkedIn_ provider to the _LinkedIn (OIDC)_ provider. Developers using the Supabase CLI to test their LinkedIn OAuth application should also update their `config.toml` to make use of the new provider:
+To support recent changes to the LinkedIn [OAuth APIs](https://learn.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow?context=linkedin%2Fcontext&tabs=HTTPS1), we will be replacing the _LinkedIn_ provider with a new _LinkedIn (OIDC)_ provider which utilizes the [Open ID Connect standard](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin-v2#validating-id-tokens). Consequently, we have disabled edits on the _LinkedIn_ provider and will be removing it effective 24th December 2023. Developers with LinkedIn OAuth Applications created prior to 1st August 2023 should create a new OAuth application [via the steps outlined above](/docs/guides/auth/social-login/auth-linkedin#create-a-linkedin-oauth-app) and migrate their credentials from the _LinkedIn_ provider to the _LinkedIn (OIDC)_ provider. Developers using the Supabase CLI to test their LinkedIn OAuth application should also update their `config.toml` to make use of the new provider:
```

Comment:
Any harm to push it maybe to Dec 28, just to not coincide within the holiday season?